packages/sanity/src/core/theme/types.ts

```diff
@@ -1,4 +1,9 @@
 import {RootTheme} from '@sanity/ui'
 
-/** @beta */
-export type StudioTheme = RootTheme
+/** @public */
+export interface StudioTheme extends RootTheme {
+  /** @internal */
+  __dark?: boolean
+  /** @internal */
+  __legacy?: boolean
```

Comment:
Could there be a case for making this not about legacy theming but rather about only having one scheme defined? If so, perhaps this could be promoted to a public property.

Reply:
Yes, probably – but that's outside of scope here.

It's `@internal` so can be renamed later.